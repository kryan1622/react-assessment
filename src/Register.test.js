import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Create from './Components/register.js';

it('render register', () => {
  const wrapper = renderer.create(<Create/>).toJSON()
  expect(wrapper).toMatchSnapshot();
})