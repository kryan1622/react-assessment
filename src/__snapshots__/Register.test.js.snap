// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render register 1`] = `
<div
  className="container"
>
  <br />
  <br />
  <h4>
     Create New User 
  </h4>
  <br />
  <br />
  <form
    onSubmit={[Function]}
  >
    <label
      className=""
    >
      Username
    </label>
    <input
      className="form-control"
      type="text"
    />
    <p
      style={
        Object {
          "color": "red",
        }
      }
    />
    <label
      className=""
    >
      Email
    </label>
    <input
      className="form-control"
      type="text"
    />
    <p
      style={
        Object {
          "color": "red",
        }
      }
    />
    <div
      className="form-group"
    >
      <label
        className=""
        htmlFor="Password"
      >
        Password
      </label>
      <input
        className="form-control"
        type="password"
      />
    </div>
    <p
      style={
        Object {
          "color": "red",
        }
      }
    />
    <div
      className="form-group"
    >
      <label
        className=""
        htmlFor="Password"
      >
        Re-enter Password
      </label>
      <input
        className="form-control"
        type="password"
      />
    </div>
    <p
      style={
        Object {
          "color": "red",
        }
      }
    />
    <p
      style={
        Object {
          "color": "red",
        }
      }
    >
      
    </p>
    <input
      type="submit"
      value="Submit"
    />
  </form>
</div>
`;
